zephyr/wrapper.c

```diff
@@ -72,18 +72,33 @@ __section(".heap_mem") static uint8_t __aligned(64) heapmem[HEAPMEM_SIZE];
 
 #else
 
-#define HEAPMEM_SHARED_SIZE	(HEAP_SYSTEM_SIZE + HEAP_RUNTIME_SIZE + \
-				HEAP_RUNTIME_SHARED_SIZE + HEAP_SYSTEM_SHARED_SIZE)
+/* TODO: heap size should be set based on linker output */
```

Comment:
Can we also comment on where we get these HEAP_SIZE numbers from (assuming default values used by Zephyr builds today based on our conversations).

Reply:
They come from the XTOS linker scripts (e.g. src/platform/apollolake/apollolake.x.in for APL):
(_bss_end - _system_heap_start)

Sure, let me add this.